src/parser.yy

```diff
@@ -626,10 +626,15 @@ sizeof_expr:
         |       SIZEOF "(" expr ")"                         { $$ = driver.ctx.make_node<ast::Sizeof>($3, @$); }
                 ;
 
+struct_field:
```

Comment:
I think our convention is that that the definitions go below where they are used.

Reply:
Thanks, just update, please review.